test/unit/snappy.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const mock = require('../tools/mock');
+const snappy = optionalRequire('snappy');
+const snappyVersion = optionalRequire('snappy/package.json').version;
+
+function optionalRequire(mod) {
+  try {
+    return require(mod);
+  } catch {
+    return false;
+  }
+}
+
+describe('Compression', function () {
+  let client;
+  /** @type {mock.MockServer} */
+  let server;
+  before(async function () {
+    if (!snappy) this.skip();
+    server = await mock.createServer();
+    client = this.configuration.newClient(`mongodb://${server.uri()}`, { compressors: 'snappy' });
+  });
+
+  after(async function () {
+    await mock.cleanup();
+    await client.close();
+  });
+
+  describe('Snappy', () => {
+    it(`should compress messages sent with snappy ${snappyVersion}`, async function () {
+      // This test should not take any longer
```

Comment:
missing the rest of the sentence?

Reply:
Oh yea, should not take any longer than 5 sec, I didn't have the comment below at first so I figured this.timeout finished my thought, whoops!